onedal/tests/utils/_dataframes_support.py

```diff
@@ -90,6 +91,10 @@ def _as_numpy(obj, *args, **kwargs):
         return dpt.to_numpy(obj, *args, **kwargs)
     if isinstance(obj, pd.DataFrame) or isinstance(obj, pd.Series):
         return obj.to_array(*args, **kwargs)
+    if hasattr(obj, "__array_namespace__"):
```

Comment:
I will take care of it on #1861 or #1813
It make sense enable it on inital functional support of Array API